src/lib/components/ProductForm.svelte

```diff
@@ -65,7 +65,9 @@
 		createdAt: new Date(),
 		updatedAt: new Date(),
 		shortDescription: '',
-		description: ''
+		description: '',
+		hideContentBefore: false,
+		hideContentAfter: false
```

Comment:
I think this should be configurable globally for products, and not on a per-product basis, no?